pkg/endpoint/policy.go

```diff
@@ -462,7 +470,7 @@ func (e *Endpoint) regenerateL3Policy(owner Owner, repo *policy.Repository, revi
 // fails for other endpoints.
 //
 // Must be called with endpoint mutex held.
-func (e *Endpoint) regeneratePolicy(owner Owner, opts models.ConfigurationMap) (bool, policy.RuleContexts, error) {
+func (e *Endpoint) regeneratePolicy(owner Owner, opts models.ConfigurationMap) (bool, bool, policy.RuleContexts, policy.RuleContexts, error) {
```

Comment:
Given the many return values of this function, document what the second boolean means and what the policy.RuleContexts represent .

Reply:
Done